index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+
+        <h1 class="header">Hey there.
+            I’m Ian.
+            <font class="text_color">I’m a software developer</font>
+        </h1>
+    </header>
+    <section>
+        <p class="paragraph"> I can help you build a product , feature or website Look through some of my work and
+            experience! If you like
+            what you see and have a project you need coded,
+            don’t hesitate to contact me.
+        </p>
+
+        <div>
+            <ul class="images">
+                <li><a href=""><img src="img/twitter.png" alt="Twitter"></a></li>
+                <li> <a href=""><img src="img/Linkedin icon.png" alt="Linkedin"></a></li>
+                <li><a href=""> <img src="img/medium.png" alt="Medium"></a></li>
+                <li><a href=""><img src="img/github.png" alt="Github"></a></li>
+                <li><a href=""><img src="img/angellist.png" alt="Angellist"></a></li>
+            </ul>
+        </div>
+        <div class="scroll">
+            <a href=""><img src="img/Disabled.png"></a>
+        </div>
+
+        <div>
+            <h1 class="header-5">My Recent Works</h1>
+            <a class="scroll" href=""><img src="img/Indicator.png"></a>
+        </div>
+
+        <div class="grid-container">
+            <div class="grid-item">
+                <div class="grid-item-top"></div>
+                <div class="grid-item-bottom">
+                    <h4 class="header-1">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="image-container">
+                        <li><a href=""><img src="img/ruby.png" alt="ruby"></a></li>
+                        <li><a href=""><img src="img/css.png" alt="css"></a></li>
+                        <li><a href=""><img src="img/javascript.png" alt="javascript"></a></li>
+                        <li><a href=""><img src="img/html.png" alt="html"></a></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
```

Comment:
- [ ] Kindly remove the dark border on the `See project` button so as to match the Figma design.

### Your Design
![your-design](https://user-images.githubusercontent.com/26597432/235254843-22ea0648-f19c-4172-b5c0-2222c36461b9.png)

### Figma Design
![temp-design](https://user-images.githubusercontent.com/26597432/235254871-d9d407fc-530c-4d9a-bdf5-e92bee008d3e.png)